### DISCUSSION TITLE
Project License on GitHub?

### DISCUSSION THREAD

**Discussion by CelestialProgrammer** [community/community#69084] [New to GitHub]
Which License should i use if i am creating a repo that would be a open software project.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
